collector.go

```diff
@@ -16,9 +16,15 @@ import (
 )
 
 var (
-	workflowRunTimeVec = prometheus.NewCounterVec(
+	workflowRunRunnerTimeVec = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
-			Name: "gha_workflow_run_time_seconds",
+			Name: "gha_workflow_run_runner_seconds",
```

Comment:
hmmm, now I see this again, the breakup of "phrases" in that metric was "gha" "workflow" "run_time" and "seconds". So I think "run_time" should be replaced with "runner".

And the other metrics should be changed similarly: s/run_time/runner/

And the metrics.md doc updated to match too.